lib/internal/bootstrap/loaders.js

```diff
@@ -40,33 +40,20 @@
 'use strict';
 
 // This file is compiled as if it's wrapped in a function with arguments
-// passed by node::LoadEnvironment()
+// passed by node::RunBootstrapping()
 /* global process, getBinding, getLinkedBinding, getInternalBinding */
-/* global debugBreak, experimentalModules, exposeInternals */
+/* global experimentalModules, exposeInternals, primordials */
 
-if (debugBreak)
-  debugger; // eslint-disable-line no-debugger
-
-const {
-  apply: ReflectApply,
-  deleteProperty: ReflectDeleteProperty,
-  get: ReflectGet,
-  getOwnPropertyDescriptor: ReflectGetOwnPropertyDescriptor,
-  has: ReflectHas,
-  set: ReflectSet,
-} = Reflect;
 const {
-  prototype: {
-    hasOwnProperty: ObjectHasOwnProperty,
-  },
-  create: ObjectCreate,
-  defineProperty: ObjectDefineProperty,
-  keys: ObjectKeys,
-} = Object;
+  Reflect,
```

Comment:
These remain mutable, so we cannot use property access on them later in the file if the intent is to ensure we get the original function for things like `Reflect.has`

Reply:
they aren't the original objects, they're null prototype namespaces with all the original methods copied

Reply:
How? Reflect is an object with a null prototype that is not available to users.

Reply:
disregard